### ISSUE TITLE
Fix NetworkSwitch perturbation tutorial

### ISSUE THREAD

**Issue by rodrigomha** [NREL-Sienna/PowerSimulationsDynamics.jl#342]
We need to explicitly state to use `using PowerNetworkMatrices` when creating Ybus. 

In addition, we should discuss if [NetworkSwitch](https://nrel-sienna.github.io/PowerSimulationsDynamics.jl/stable/api/public/#PowerSimulationsDynamics.NetworkSwitch) should ONLY use a SparseMatrix or not. From a user perspective we should allow the user to use directly a dense matrix if they want.

In any case, we need to update the tutorial of Perturbations since it is using Dense Matrices: https://nrel-sienna.github.io/PowerSimulationsDynamics.jl/stable/perturbations/

**Comment by jd-lara** [comment]
I don't think this is a bug. We have always assumed that the user would pass a Bus object which is by default sparse.
